build.zig

```diff
@@ -195,7 +195,7 @@ pub fn build(b: *std.Build) !void {
         .HAVE_SYS_UTSNAME_H = modernUnix,
         .HAVE_SYS_WAIT_H = false, // unused
         .HAVE_TERMIOS_H = modernUnix,
-        .HAVE_WORKING_LIBINTL = isLinux,
+        .HAVE_WORKING_LIBINTL = isLinux and !t.isMuslLibC(),
```

Comment:
hmm, perhaps it would be possible to phrase this positively, like `HAVE_WORKING_LIBNTL = t.isGnuLibC()` ?

Eventually, we need to either (1) find an universal solution for translations (a self-contained c or zig library which just works on all platforms) or (2) drop translations entirely...